HermesProxy/World/Enums/PlayerDefines.cs

```diff
@@ -140,7 +140,17 @@ public enum RestState
         Normal = 0x02,
     };
 
-    public enum ChatFlags
+    [Flags]
+    public enum ChatFlagsVanilla
+    {
+        None = 0,
+        AFK = 1,
+        DND = 2,
+        GM = 3,
+    }
```

Comment:
Looks like the same values to me as `ChatFlagsModern`, there is no need to create a new enum here.